src/DynamoCoreWpf/UI/GuidedTour/Step.cs

```diff
@@ -235,6 +236,12 @@ where automation.Name.Equals(calculateLibraryFuncName)
                
 
             stepUIPopup.IsOpen = true;
+
+            var nextButton = stepUIPopup.FindName(NextButton) as Button;
+            if (nextButton != null) 
+            {
+                nextButton.Focus();
```

Comment:
So I assume once the button gets focus, we can use arrow keys for these?

Reply:
yeah, that's correct.